app/models/setting/provisioning.rb

```diff
@@ -20,7 +20,7 @@ def self.local_boot_labels
     'usb*',
     'vnet*',
     'macvtap*',
-    '_vdsmdummy_',
```

Comment:
Was the old one unused? From the subject I expected it to only add, not replace one.

Reply:
```
Additional info:
Current interface output of a RHV-Linux hypervisor with OVS enabled:

# ip l |egrep "vdsmdummy|br-int"
7: br-int: <BROADCAST,MULTICAST> mtu 1442 qdisc noop state DOWN mode DEFAULT group default qlen 1000
40: ;vdsmdummy;: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN mode DEFAULT group default qlen 1000
```

Looks like the one with underscores was wrong and the other was missing

Reply:
I also think so: https://github.com/oVirt/ovirt-engine/search?q=vdsmdummy&type=Code

Reply:
Looks like you're right. Looking at https://github.com/oVirt/vdsm/commit/a8708b8a8d7a1e7c5abde2b6b1cb8ad6c4555ddd#diff-4fda6a624b7ea6ccc1842e181c60b171R46 (dated in 2012) I think it's safe to assume the underscores version never existed.